tools/run_tests/run_tests.py

```diff
@@ -57,8 +57,8 @@ def __init__(self, config, environ={}):
     self.allow_hashing = (config != 'gcov')
     self.environ = environ
 
-  def job_spec(self, binary, hash_targets):
-    return jobset.JobSpec(cmdline=[binary],
+  def job_spec(self, cmdline, hash_targets):
```

Comment:
Yes; please do this one method now.

I look forward to the entire file but agree that it would be out-of-scope for this change.